Teacher success product design histories (#1665)
DFE-Digital/bat-design-history: app/posts/teacher-success/2025-07-01-teacher-success-traffic_acquisition.md

---
title: Get ready to teach traffic driving approach 
description: How we are driving trainees to Teacher Success 
date: 2025-07-01
tags:
  - withdrawals
  - training
  - training outcomes
---

The Teacher Success team is delivering a digital companion service in time for the 2025/26 academic year, helping to set better expectations and signposting support for trainee teachers. At point of launch the service will be titled 'Get ready to teach'.  

## Positioning the teacher success service

> A key aim of get ready to teach is to continue the journey of inspiration and encouragement established by the Get into Teaching (GIT) service. The service will use enough imagery and design cues from GIT to make this continuity clear to users, whilst having a brand identity that makes it clearly a new phase of the journey. In future, it is expected that Get ready to teach will provide a personalisation gateway into comparable supporting content and services for early career teachers and potentially more senior practitioners.

Reviewer: ```suggestion
A key aim of Get ready to teach is to continue the journey of inspiration and encouragement established by the Get into Teaching (GIT) service. The service will use enough imagery and design cues from GIT to make this continuity clear to users, whilst having a brand identity that makes it clearly a new phase of the journey. In future, it is expected that Get ready to teach will provide a personalisation gateway into comparable supporting content and services for early career teachers and potentially more senior practitioners.
```

> Get ready to teach will, be a separate website to GIT, so we needed to ensure that the two services do not compete for organic search traffic or bid on the same keywords for any paid search.

Reviewer: ```suggestion
Get ready to teach will be a separate website to GIT, so we needed to ensure that the 2 services do not compete for organic search traffic or bid on the same keywords for any paid search.
```

## Targeted comms vs mass market traffic acquisition

> GIT is at the top of the conversion funnel and is concerned with attracting candidates into Initial teacher training (ITT), so a search engine results page (SERP, eg Google, Bing) strategy is appropriate for that service. However, get ready to teach is concerned in the first instance with targeting an MVP cohort (around 7,000 trainees of an estimated 20,000) with timely signposting to content appropriate to the stage of the academic year.  

Reviewer: ```suggestion
GIT is at the top of the conversion funnel and is concerned with attracting candidates into initial teacher training (ITT), so a search engine results page (SERP, eg Google, Bing) strategy is appropriate for that service. However, get ready to teach is concerned in the first instance with targeting an MVP cohort (around 7,000 trainees of an estimated 20,000) with timely signposting to content appropriate to the stage of the academic year.  
```

> GIT is at the top of the conversion funnel and is concerned with attracting candidates into Initial teacher training (ITT), so a search engine results page (SERP, eg Google, Bing) strategy is appropriate for that service. However, get ready to teach is concerned in the first instance with targeting an MVP cohort (around 7,000 trainees of an estimated 20,000) with timely signposting to content appropriate to the stage of the academic year.  

Reviewer: ```suggestion
GIT is at the top of the conversion funnel and is concerned with attracting candidates into Initial teacher training (ITT), so a search engine results page (SERP, for example, Google or Bing) strategy is appropriate for that service. However, get ready to teach is concerned in the first instance with targeting an MVP cohort (around 7,000 trainees of an estimated 20,000) with timely signposting to content appropriate to the stage of the academic year.  
```

> GIT is at the top of the conversion funnel and is concerned with attracting candidates into Initial teacher training (ITT), so a search engine results page (SERP, eg Google, Bing) strategy is appropriate for that service. However, get ready to teach is concerned in the first instance with targeting an MVP cohort (around 7,000 trainees of an estimated 20,000) with timely signposting to content appropriate to the stage of the academic year.  

Reviewer: ```suggestion
GIT is at the top of the conversion funnel and is concerned with attracting candidates into Initial teacher training (ITT), so a search engine results page (SERP, eg Google, Bing) strategy is appropriate for that service. However, Get ready to teach is concerned in the first instance with targeting an MVP cohort (around 7,000 trainees of an estimated 20,000) with timely signposting to content appropriate to the stage of the academic year.  
```

## Deciding not to index pages

> Given these two factors, we decided that not indexing our web pages would be the best approach for MVP and possibly beyond. Non indexed pages are public but cannot be found on SERP. This means that our pages will not compete for prominence with GIT and we will also be able to more directly measure the effectiveness of our email comms at driving traffic to the right page at the right time.

Reviewer: ```suggestion
Given these 2 factors, we decided that not indexing our web pages would be the best approach for MVP and possibly beyond. Non indexed pages are public but cannot be found on SERP. This means that our pages will not compete for prominence with GIT and we will also be able to more directly measure the effectiveness of our email comms at driving traffic to the right page at the right time.
```
